### ISSUE TITLE
Missing Atribute.

### ISSUE THREAD

**Issue by GIAPB** [jcabi/jcabi-github#1270]
Everything works while in eclipse. The only problem is,
when exported, the console throws this error: 
```C:\Users\matthew\Desktop>java -jar GIAPB.jar
SLF4J: Failed to load class "org.slf4j.impl.StaticLoggerBinder".
SLF4J: Defaulting to no-operation (NOP) logger implementation
SLF4J: See http://www.slf4j.org/codes.html#StaticLoggerBinder for further detail
s.
Exception in thread "main" java.lang.ExceptionInInitializerError
        at xyz.lexium.giapb.GIAPB.main(GIAPB.java:22)
Caused by: java.lang.IllegalArgumentException: Attribute 'JCabi-Version' not fou
nd in MANIFEST.MF file(s) among 3 other attribute(s): ["Class-Path", "Main-Class
", "Manifest-Version"]
        at com.jcabi.manifests.Manifests.read(Manifests.java:274)
        at com.jcabi.github.RtGithub.<clinit>(RtGithub.java:82)
        ... 1 more

C:\Users\matthew\Desktop>
```

**Comment by 0crat** [comment]
@yegor256 please, pay attention to this issue

**Comment by yegor256** [comment]
@GIAPB how do you package your app? It seems that you delete that valuable `META-INF/MANIFEST.MF` that is supposed to be packaged together with all classes from jcabi-github.jar

**Comment by xXXIMMATTHEWXXx** [comment]
It in in xyz.lexium.giapb. Nothig else is changed. Just compiled via eclipse

**Comment by yegor256** [comment]
@xXXIMMATTHEWXXx I'm afraid that "compile via eclipse" is what broke the structure of the JAR file. Try to package with Maven.

**Comment by xXXIMMATTHEWXXx** [comment]
Thanks

**Comment by amihaiemil** [comment]
@GIAPB  @xXXIMMATTHEWXXx The problem is that the library reads some attributes from its Manifest.MF file. This causes troubles when it is packaged in a ``.war`` for instance. You can solve this issue with ``maven-assembly-plugin``. Here's how I did it: https://github.com/jcabi/jcabi-github/issues/1198#issuecomment-225122003 .

There is also an issue open, to change this mechanism; https://github.com/jcabi/jcabi-github/issues/1246

**Comment by amihaiemil** [comment]
This has been fixed in [release 0.37](https://github.com/jcabi/jcabi-github/issues/1373).

**Comment by 0crat** [comment]
Job `gh:jcabi/jcabi-github#1270` is not assigned, can't get performer
